foundational-cxx-support-matrix.md

```diff
@@ -10,14 +10,14 @@
 | MSVC            | >= 2022                | 2024-04-29   | 2027-01-12  |
 | Apple Clang     | >= 12                  | 2022-07-01   | |
 | Android NDK API | == 21                  | 2022-09-18   | |
-| Alpine          | >= 3.16                | 2024-01-02   | 2024-06-01 |
+| Alpine          | >= 3.17                | 2024-05-23   | 2024-11-22 |
 | Debian          | >= 10                  | 2022-07-01   | 2024-07-01 |
 | Fedora          | >= 39                  | 2024-01-02   | 2024-12-01 |
 | openSUSE        | >= Leap 15.5           | 2024-01-02   | 2025-01-01 |
 | Ubuntu LTS      | >= 20.04               | 2023-06-01   | 2025-04-02 |
 | RHEL            | >= 7 [^rhel-7]         | 2022-07-01   | 2024-07-01 |
 | CentOS          | 7 [^rhel-7]            | 2022-07-01   | 2024-07-01 |
-| RockyLinux      | 8                      | 2022-07-01   | 2024-06-01 |
+| RockyLinux      | 8                      | 2022-07-01   | 2029-05-31 |
```

Comment:
Please revert this change. We do not support development on distros during the security-patches only period.